doc/quickstart.rst

```diff
@@ -16,8 +16,9 @@ Pip can install ``suricata-update`` globally making it available to
 all users or it can install ``suricata-update`` into your home
 directory for use by your user.
 
-.. note:: At some point ``suricata-update`` should be bundled with
-          Suricata avoid the need for a separate installation.
+.. note:: Version 4.1+ from the Ubuntu PPA is bundled with
```

Comment:
Hi @utkonos !
Thanks for your first contribution to the project. :)
I was wondering if we should drop the "from Ubuntu PPA" part as it was bundled for every distribution. https://suricata-ids.org/2018/11/06/suricata-4-1-released/

Reply:
> Hi @utkonos !
> Thanks for your first contribution to the project. :)
> I was wondering if we should drop the "from Ubuntu PPA" part as it was bundled for every distribution. https://suricata-ids.org/2018/11/06/suricata-4-1-released/

That's great. Even better.